client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClientConfig.java

```diff
@@ -23,4 +23,51 @@
       "schema.registry.basic.auth.user.info";
   public static final String USER_INFO_CONFIG =
       "basic.auth.user.info";
+
+  public static final String SSL_KEYSTORE_LOCATION_CONFIG = "ssl.keystore.location";
```

Comment:
is there a way we can avoid duplicating those configs and somehow reuse existing configdefs?

Reply:
The existing config defs were only on the server side. Moreover, on the client side, the configs need namespacing as well.

Reply:
Can we extract the configs used across clients and server into a single utility with various methods for adding config sets to a ConfigDef?

Reply:
@mageshn, I used AK security configs client side in Elastic connector: https://github.com/confluentinc/kafka-connect-elasticsearch/pull/278

Reply:
@rhauch I already extracted the configs to the client side and the server side also uses the same. @cyrusv unfortunately, we can't use the AK configdef as-is since SR uses the config-def from rest-utils

Reply:
I don't understand why config-def from rest-utils precludes using AK configs?

Reply:
@mageshn Any update? Can you fix the conflict as well?